org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/StandardPreferenceManager.java

```diff
@@ -102,24 +107,52 @@ public void update(Preferences preferences) {
 		try {
 			Settings mavenSettings = MavenPlugin.getMaven().getSettings();
 			boolean oldDisableTest = false;
+			String multiModuleProjectDirectory = null;
+			String systemMmpd = System.getProperty(MAVEN_MULTI_MODULE_PROJECT_DIRECTORY);
```

Comment:
Is this provided so clients can set this prior to project import via. vmargs ?

Reply:
Yes, it is.